pytket/pytket/circuit/__init__.py

```diff
@@ -72,7 +72,7 @@ def overload_add_wasm(
 ) -> Circuit:
     """Add a classical function call from a wasm file to the circuit.
     \n\n:param funcname: name of the function that is called
-    \n:param filehandler: wasm file handler to identify the wasm file
+    \n:param filehandler: wasm file or module handler to identify the wasm module
```

Comment:
Shouldn't we change the type annotation to a `Union` if both types are accepted (not sure why mypy didn't complain)?

Reply:
Like I mentioned in the OP, `WasmFileHandler` is a subclass of `WasmModuleHandler`. If we are accepting the superclass, any of its subclasses is acceptable.

Reply:
But if you prefer the Union for better documentation from types, I am happy to make that change, let me know.